test/apps/iodemo/io_demo.cc

```diff
@@ -16,6 +16,8 @@
 #include <unistd.h>
 #include <cstdlib>
 #include <ctime>
+#include <csignal>
+#include <cerrno>
```

Comment:
still needed?

Reply:
yes, we are printing error if `sigaction` faield

Reply:
ok